core-lib/TestSuite/Minitest.ns

```diff
@@ -441,12 +441,12 @@ OTHER DEALINGS IN THE SOFTWARE. *)
               exClass:: (ClassMirror reflecting: e) classObject.
               exClass == TestFailureException ifTrue: [
                 promisePair resolve: (testContextInstance
-                  ifNil:    [ TestFailure case: self description: ex messageText]
-                  ifNotNil: [:it | it createFailureResultFor: self description: ex messageText])
+                  ifNil:    [ TestFailure case: self description: e messageText]
+                  ifNotNil: [:it | it createFailureResultFor: self description: e messageText])
               ] ifFalse: [
                 promisePair resolve: (testContextInstance
-                  ifNil:    [ TestError case: self exception: ex ]
-                  ifNotNil: [:it | it createErrorResultFor: self exception: ex ])
+                  ifNil:    [ TestError case: self exception: e ]
+                  ifNotNil: [:it | it createErrorResultFor: self exception: e ])
```

Comment:
There is no ex variable in that scope, only e. Instead of fixing the usage I can rename e to ex if you prefer.

Reply:
Ahhhh. No, the change is good. My comment just shows that it would help me to get more context for such changes (possibly by separate maintenance PRs, added tests, ...)

I mean, it's already good that you have a separate commit for the timeout stuff. But it's unfortunate that it doesn't come with any testing, any any comment for the rational for the changes.